hashicorp/boundary-ui
# Icu 15713 address deprecation warning for dropdown interactive element

# Description
This pr fixes the deprecation warning for the dropdown interactive element. 

Note: I used codemod for some of the files, but I had to manually update the rest where the element is wrapped with a conditional statement since the codemod currently does not handle this case. ticket [here](https://hashicorp.atlassian.net/browse/HDS-4151) for that


   



## Screenshots (if appropriate)

## How to Test

1. click on any page that has the dropdown interactive element (auth-methods->oidc->click on change state component)
2. open the console, you should not see the deprecation warning 

## Checklist
